lib/ed25519PublicKeyNode12.js

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {asn1, oids, util: {ByteBuffer}} = require('node-forge');
+const {createPublicKey} = require('crypto');
+
+exports.create = ({publicKeyBytes}) => createPublicKey({
+  key: exports._derEncode({publicKeyBytes}),
+  format: 'der',
+  type: 'spki'
+});
+
+exports._derEncode = ({publicKeyBytes}) => {
```

Comment:
Seems like this should be in forge.

Reply:
takes FOREVER to get stuff committed to forge!  :)